src/assets/scss/_gallery.scss

```diff
@@ -0,0 +1,86 @@
+//Gallery section
+
+.gallery {
+    display: flex;
+    align-items: center;
+    padding: 8rem 10rem;
+    flex-wrap: wrap;
+  
+    &--btns {
+      display: flex;
+      width: 19rem;
+      justify-content: space-between;
+      margin: 2rem 0;
+    }
+    &--btn1,
+    &--btn2,
+    &--btn3 {
+      height: 0.5rem;
+      width: 4rem;
+      outline: none;
+      border-style: none;
+      background-color: $grey-btn-color;
+
+      &:hover,
+      &:active,
+      &:focus {
+        background-color: $active-grey-btn-color;
+        outline: none;
+        border-style: none;
+      } 
+    }
+  
+    
+    &--element {
+      list-style: none;
+      padding: none;
+      justify-content: space-between;
+    }
+
+    &--photo {
+      width: 50rem;
+      height: 27rem;
+      object-fit: cover;
+      padding-left: 7rem;
+    }
+  
+    &--text {
+      flex-direction: column;
+      width: 19rem;
+    }
+
+    &--title {
+      font-size: 2rem;
+      font-family: $serif;
+      padding-bottom: 0.5rem;
+    }
+  
+    &--project-type {
+      font-family: $sans-serif;
+      font-size: 1.5rem;
+      padding-bottom: 1.5rem;
+      font-weight: 300;
+    }
+  
+    &--description {
+      font-family: $sans-serif;
+      font-weight: 300;
+    }
+       
+  
+    &--element1 {
```

Comment:
tutaj zrób sobie klasę __element, która ma display: none,  + __element--visible z display: flex, 

Z tym, że display się nie animuje, jeśli będziesz chciała animację zrobić to będzie trzeba dodać opacity: 0, visibility: hidden + albo position: absolute (i wtedy wszystkie te elementy na absolutach) albo coś innego pokombinować :)